liquidity/liquidity.go

```diff
@@ -387,13 +415,15 @@ func (m *Manager) autoloop(ctx context.Context) error {
 			swap.DestAddr = m.params.DestAddr
 		}
 
-		loopOut, err := m.cfg.LoopOut(ctx, &swap)
+		err := m.dispatchStickyLoopOut(
```

Comment:
IMO the backoff logic should either be configurable / and or be able to be disabled, as the backoff amount might collide with the rules we have set. I'm not sure If a 3 times reduced swap would be favorable

Reply:
The previous behavior is a sub-set to this PR. With no failures present, the behavior will be identical whether we have backoff implemented or not.

> I'm not sure If a 3 times reduced swap would be favorable

What would "favorable" mean here?

Reply:
The user defined restrictions for swaps (w.r.t to fee percentages etc) are checked for every individual loop that we dispatch, regardless of if we have applied an amount backoff to it.